neofetch

```diff
@@ -9644,7 +9657,41 @@ EOF
     }
 }
 
+get_longest_label() {
+    # Parse the user's configuration file for the longest label
+    longest_label_length=0
+	{
+		while read -r line; do
+			# By splitting the string on quotes we ensure that the first field is always
+			# extraneous info we don't care about. If the line doesn't have what we want
+			# then there is no second field. If it does, then the second field is the
+			# label of that line of info, and the longest one is the longest label.
+			IFS='\"' read -r -a this_line_labels <<< "$line"
+			IFS=' ' read -r -a this_line_spaces <<< "$line"
+			if [[ ${this_line_spaces[0]} != "#" ]]; then
+				if [[ ${#this_line_labels[1]} -gt $longest_label_length ]]; then
+					longest_label_length=${#this_line_labels[1]}
+				fi
+			fi
+			if [[ "$line" == "}" ]]; then
+				break
+			fi
+		done
+	} < "${config_file}"
+	echo "$longest_label_length"
+}
+
+get_offset() {
+    # Longest string. I have no idea why this magic number 18
+    # exists, but it works for /every/ ascii art I tried.
+    longest_string_offset=$((18 + $(get_longest_label) + ${#separator}))
+    offset=$((text_padding - longest_string_offset))
+    offset=$((text_padding - offset))
+    echo $offset
```

Comment:
Remove this line.

Reply:
Can you clarify? This looks like the whole function -- did you mean the comment?

Reply:
I've removed the comment in a new force-push -- feel free to unresolve if necessary.